swatch-tally/deploy/clowdapp.yaml

```diff
@@ -122,6 +123,29 @@ parameters:
   - name: INVENTORY_SECRET_KEY_NAME_PREFIX
     value: ''
 
+  - name: TENANT_TRANSLATOR_HOST
+    value: "apicast.3scale-dev.svc.cluster.local"
+  - name: TENANT_TRANSLATOR_PORT
+    value: '8891'
+  - name: POPULATOR_LOG_FORMAT
+    value: cloudwatch
+  - name: ORG_ID_POPULATOR_IMAGE
+    value: quay.io/cloudservices/tenant-utils
+  - name: ORG_ID_POPULATOR_IMAGE_TAG
+    value: latest
+  - name: ORG_ID_POPULATOR_CRON_CPU_REQUEST
+    value: 50m
+  - name: ORG_ID_POPULATOR_CRON_MEMORY_REQUEST
+    value: 512Mi
+  - name: ORG_ID_POPULATOR_CRON_CPU_LIMIT
+    value: 300m
+  - name: ORG_ID_POPULATOR_CRON_MEMORY_LIMIT
+    value: 1Gi
```

Comment:
This suggests it's a CronJob / has a schedule, please remove `_CRON` from these vars.

Reply:
DONE